src/games/strategy/engine/data/export/RulesAttachmentExporter.java

```diff
@@ -134,11 +135,9 @@ private String mTurnsHandler(final Field field, final IAttachment attachment) {
     try {
       oValue = field.get(attachment);
     } catch (final IllegalArgumentException e) {
```

Comment:
More work is needed than simple removal, would need to verify that the code still functions the same, which woudl involve looking at Field and when exactly it would throw illegal arg.